src/forklift/core.py

```diff
@@ -398,8 +398,14 @@ def _is_naughty_field(field):
     '''
     #: global id's do not export to file geodatabase
     #: removes objectid_ which is created by geoprocessing tasks and wouldn't be in destination source
-    #: TODO: Deal with possibility of OBJECTID_* being the OIDFieldName
-    return 'SHAPE' in field.upper() or field.upper() in ['GLOBAL_ID', 'GLOBALID'] or field.startswith('OBJECTID')
+    skip_fields = ['GLOBAL_ID', 'GLOBALID']
+
+    if describe is not None:
+        for prop in ['shapeFieldName', 'lengthFieldName', 'OIDFieldName']:
+            if prop in describe:
+                skip_fields.append(describe[prop].upper())
```

Comment:
**thought (non-blocking)**: in the rest of the project we seem to use `.lower()` for our case insensitive checks.

Reply:
def3ffa